apps/condo/domains/ticket/constants/emoji.js

```diff
@@ -7,6 +7,14 @@ const EMOJI = {
     ROCKET: '🚀',
 }
 
+const EMOJI_IMAGES = {
+    BROKEN_HEART: '/emoji/broken-heart.png',
+    SNAIL: '/emoji/snail.png',
+    TROPHY: '/emoji/trophy.png',
+    ROCKET: '/emoji/rocket.png',
```

Comment:
we really need png?

why not use jpeg or svg? I think it might weigh less and look better